packages/components/package.json

```diff
@@ -27,7 +26,6 @@
   "module": "dist/web3-ui-components.esm.js",
   "types": "dist/web3-ui-components.cjs.d.ts",
   "dependencies": {
-    "@babel/runtime": "^7.16.3",
```

Comment:
without `@babel/runtime` the regenerator runtime won't work https://www.runpkg.com/?@web3-ui/hooks@0.5.0/dist/web3-ui-hooks.esm.js#1 the bundles aren't too ideal in terms of build composition 😅

Reply:
I see the helpers inlined in the build output locally?

Reply:
oh weird, this used to bug out before

Reply:
I see what you're saying locally. That… doesn't make any sense.

Reply:
Yes, it caught me off-guard as well when I noticed 😅 judging from their issues it feels "intended" but then again I always try to avoid async await in my OSS projects so not too familiar with the matter

Reply:
I wonder what would cause this. This is similar to the other comment in that I've never had to use this before in any other projects and haven't run into issues. 

I see a few things referencing runtime-related things in this issue: https://github.com/preconstruct/preconstruct/issues/297#issuecomment-690964802

But there's nothing that jumps out to me that this should be required. I feel like we should open a new issue over there but I'm not even sure what to report exactly 😆